geomstats/geometry/invariant_metric.py

```diff
@@ -49,8 +50,9 @@ def __init__(self, space, metric_mat_at_identity=None, left=True):
         self.metric_mat_at_identity = metric_mat_at_identity
         self.left = left
 
-    def reshape_metric_matrix(self):
-        """Reshape diagonal metric matrix to a symmetric matrix of size n.
+    @property
+    def reshaped_metric_matrix(self):
+        """Diagonal metric matrix reshaped to a symmetric matrix of size n.
```

Comment:
Does the rule "Docstring starts with verb at infinitive" not apply for @property ?